### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by ldilalla-HC**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,84 +1,173 @@
...
+and signing APIs will now include the root CA certificate if the mount is
+aware of it. [[GH-15155](https://github.com/hashicorp/vault/pull/15155)]
+* secrets/pki: existing Delete Root API (pki/root) will now delete all issuers
+and keys within the mount path. [[GH-15004](https://github.com/hashicorp/vault/pull/15004)]
+* secrets/pki: existing Generate Root (pki/root/generate/:type),
+Set Signed Intermediate (/pki/intermediate/set-signed) APIs will
+add new issuers/keys to a mount instead of warning that an existing CA exists [[GH-14975](https://github.com/hashicorp/vault/pull/14975)]
+* secrets/pki: the signed CA certificate from the sign-intermediate api will now appear within the ca_chain
+response field along with the issuer's ca chain. [[GH-15524](https://github.com/hashicorp/vault/pull/15524)]
 * ui: Upgrade Ember to version 3.28 [[GH-14763](https://github.com/hashicorp/vault/pull/14763)]
```
Comment (mladlow) [inline CHANGELOG.md:38]:
```suggestion
* ui: Upgrade Ember to version 3.28 [[GH-14763](https://github.com/hashicorp/vault/pull/14763)]

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,84 +1,173 @@
...
+* **Print minimum required policy for any command**: The global CLI flag `-output-policy` can now be used with any command to print out the minimum required policy HCL for that operation, including whether the given path requires the "sudo" capability. [[GH-14899](https://github.com/hashicorp/vault/pull/14899)]
+* **Snowflake Database Plugin**: Adds ability to manage RSA key pair credentials for dynamic and static Snowflake users. [[GH-15376](https://github.com/hashicorp/vault/pull/15376)]
+* **Transit BYOK**: Allow import of externally-generated keys into the Transit secrets engine. [[GH-15414](https://github.com/hashicorp/vault/pull/15414)]
 * nomad: Bootstrap Nomad ACL system if no token is provided [[GH-12451](https://github.com/hashicorp/vault/pull/12451)]
 * storage/dynamodb: Added `AWS_DYNAMODB_REGION` environment variable. [[GH-15054](https://github.com/hashicorp/vault/pull/15054)]
+* storage/raft (enterprise): Enable autopilot automated upgrades and redundancy zones.
 
 IMPROVEMENTS:
 
+* :core/managed-keys (enterprise): Allow configuring the number of parallel operations to PKCS#11 managed keys.
```
Comment (mladlow) [inline CHANGELOG.md:56]:
```suggestion
* core/managed-keys (enterprise): Allow configuring the number of parallel operations to PKCS#11 managed keys.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,84 +1,173 @@
...
 * licensing (enterprise): Remove support for stored licenses and associated `sys/license` and `sys/license/signed`
 endpoints in favor of [autoloaded licenses](https://www.vaultproject.io/docs/enterprise/license/autoloading).
+* physical/cockroachdb: Change underlying driver library from [lib/pq](https://github.com/lib/pq) to [pgx](https://github.com/jackc/pgx) [[GH-15343](https://github.com/hashicorp/vault/pull/15343)]
+* physical/postgres: Change underlying driver library from [lib/pq](https://github.com/lib/pq) to [pgx](https://github.com/jackc/pgx) [[GH-15343](https://github.com/hashicorp/vault/pull/15343)]
 * replication (enterprise): The `/sys/replication/performance/primary/mount-filter` endpoint has been removed. Please use [Paths Filter](https://www.vaultproject.io/api-docs/system/replication/replication-performance#create-paths-filter) instead.
+* secret/pki: Remove unused signature_bits parameter from intermediate CSR generation; this parameter doesn't control the final certificate's signature algorithm selection as that is up to the signing CA [[GH-15478](https://github.com/hashicorp/vault/pull/15478)]
```
Comment (mladlow) [inline CHANGELOG.md:19]: There's a TON of PKI changelog entries in CHANGES. I'm not sure if they should all be CHANGES or if some should be improvements, or features, or a different type.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,84 +1,173 @@
...
-* api/command: Global -output-policy flag to determine minimum required policy HCL for a given operation [[GH-14899](https://github.com/hashicorp/vault/pull/14899)]
+* **Print minimum required policy for any command**: The global CLI flag `-output-policy` can now be used with any command to print out the minimum required policy HCL for that operation, including whether the given path requires the "sudo" capability. [[GH-14899](https://github.com/hashicorp/vault/pull/14899)]
+* **Snowflake Database Plugin**: Adds ability to manage RSA key pair credentials for dynamic and static Snowflake users. [[GH-15376](https://github.com/hashicorp/vault/pull/15376)]
+* **Transit BYOK**: Allow import of externally-generated keys into the Transit secrets engine. [[GH-15414](https://github.com/hashicorp/vault/pull/15414)]
 * nomad: Bootstrap Nomad ACL system if no token is provided [[GH-12451](https://github.com/hashicorp/vault/pull/12451)]
 * storage/dynamodb: Added `AWS_DYNAMODB_REGION` environment variable. [[GH-15054](https://github.com/hashicorp/vault/pull/15054)]
+* storage/raft (enterprise): Enable autopilot automated upgrades and redundancy zones.
```
Comment (mladlow) [inline CHANGELOG.md:52]:
```suggestion
* **Autopilot Improvements (Enterprise)**: Autopilot on Vault Enterprise now supports automated upgrades and redundancy zones when using integrated storage.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,84 +1,173 @@
...
 
 CHANGES:
 
 * auth/aws: Add RoleSession to DisplayName when using assumeRole for authentication [[GH-14954](https://github.com/hashicorp/vault/pull/14954)]
 * auth: Remove support for legacy MFA
 (https://www.vaultproject.io/docs/v1.10.x/auth/mfa) [[GH-14869](https://github.com/hashicorp/vault/pull/14869)]
 * core: A request that fails path validation due to relative path check will now be responded to with a 400 rather than 500. [[GH-14328](https://github.com/hashicorp/vault/pull/14328)]
-* core: Bump Go version to 1.17.9. [[GH-go-ver-1110](https://github.com/hashicorp/vault/pull/go-ver-1110)]
+* core: Bump Go version to 1.17.11. [[GH-go-ver-1110](https://github.com/hashicorp/vault/pull/go-ver-1110)]
+* database/postgres: Change underlying driver library from [lib/pq](https://github.com/lib/pq) to [pgx](https://github.com/jackc/pgx) [[GH-15343](https://github.com/hashicorp/vault/pull/15343)]
```
Comment (mladlow) [inline CHANGELOG.md:12]: @kalafut do you think it's worth consolidating all these similar lines to reduce the total number of entries in the CHANGES section?

Comment (kalafut) [reply CHANGELOG.md:12]: Yeah, that makes sense to me. Maybe to something like:

```
* database & storage: Change underlying driver library.... This change affects Redshift & Postgres database secrets engines, and CockroachDB & Postgres storage engines.
```

File: CHANGELOG.md (hunk 3 above)
Comment (ldilalla-HC) [reply CHANGELOG.md:16]: we will take a look for the GA version

File: CHANGELOG.md [hunk 6]
```diff
@@ -762,6 +940,18 @@ of dirty pages in the merkle tree at time of checkpoint creation. [[GH-2093](htt
 * ui: update bar chart when model changes [[GH-12622](https://github.com/hashicorp/vault/pull/12622)]
 * ui: updating database TTL picker help text. [[GH-12212](https://github.com/hashicorp/vault/pull/12212)]
 
+## 1.8.12
+### June 9,2022
```
Comment (mladlow) [inline CHANGELOG.md:944]:
```suggestion
### June 10, 2022
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -394,6 +541,37 @@ operation for upgraded configurations with a `root_password_ttl` of zero. [[GH-1
 * ui: Removes ability to tune token_type for token auth methods [[GH-12904](https://github.com/hashicorp/vault/pull/12904)]
 * ui: trigger token renewal if inactive and half of TTL has passed [[GH-13950](https://github.com/hashicorp/vault/pull/13950)]
 
+## 1.9.7
+### June 9, 2022
```
Comment (mladlow) [inline CHANGELOG.md:545]:
```suggestion
### June 10, 2022
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -87,30 +174,90 @@ BUG FIXES:
...
 * ui: Fixes edit auth method capabilities issue [[GH-14966](https://github.com/hashicorp/vault/pull/14966)]
 * ui: Fixes issue logging in with OIDC from a listed auth mounts tab [[GH-14916](https://github.com/hashicorp/vault/pull/14916)]
+* ui: Revert using localStorage in favor of sessionStorage [[GH-15769](https://github.com/hashicorp/vault/pull/15769)]
 * ui: fix firefox inability to recognize file format of client count csv export [[GH-15364](https://github.com/hashicorp/vault/pull/15364)]
+* ui: fix form validations ignoring default values and disabling submit button [[GH-15560](https://github.com/hashicorp/vault/pull/15560)]
 * ui: fix search-select component showing blank selections when editing group member entity [[GH-15058](https://github.com/hashicorp/vault/pull/15058)]
 * ui: masked values no longer give away length or location of special characters [[GH-15025](https://github.com/hashicorp/vault/pull/15025)]
 
+## 1.10.4
+### June 9, 2022
```
Comment (mladlow) [inline CHANGELOG.md:219]:
```suggestion
### June 10, 2022
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,84 +1,171 @@
 
-## 1.11.0
-### Unreleased
+## 1.11.0-rc1
+### June 9, 2022
```
Comment (mladlow) [inline CHANGELOG.md:3]:
```suggestion
### June 10, 2022
```

File: CHANGELOG.md (hunk 5 above)
Comment (ldilalla-HC) [reply CHANGELOG.md]: made the suggested changes

**Review by mladlow — APPROVED** [review]

**Pull request merged**